From akpm@osdl.org Fri Feb  3 01:46:31 2006
Message-Id: <200602030946.k139kIW6002410@shell0.pdx.osdl.net>
Subject: PCI: fix the x86 pci domain support fix
To: greg@kroah.com
Cc: akpm@osdl.org, apw@shadowen.org, jgarzik@pobox.com, mbligh@aracnet.com
From: akpm@osdl.org
Date: Fri, 03 Feb 2006 01:45:58 -0800


From: Andrew Morton <akpm@osdl.org>

x86_64 needs this declared too..

arch/x86_64/pci/../../i386/pci/fixup.c: In function `pci_fixup_i450nx':
arch/x86_64/pci/../../i386/pci/fixup.c:29: error: `pci_default_sysdata' undeclared (first use in this function)
arch/x86_64/pci/../../i386/pci/fixup.c:29: error: (Each undeclared identifier is reported only once
arch/x86_64/pci/../../i386/pci/fixup.c:29: error: for each function it appears in.)
arch/x86_64/pci/../../i386/pci/fixup.c: In function `pci_fixup_i450gx':
arch/x86_64/pci/../../i386/pci/fixup.c:47: error: `pci_default_sysdata' undeclared (first use in this function)

Cc: Andy Whitcroft <apw@shadowen.org>
Cc: Jeff Garzik <jgarzik@pobox.com>
Cc: "Martin J. Bligh" <mbligh@mbligh.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 include/asm-x86_64/pci.h |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/include/asm-x86_64/pci.h
+++ gregkh-2.6/include/asm-x86_64/pci.h
@@ -10,6 +10,7 @@ struct pci_sysdata {
 	int		domain;		/* PCI domain */
 	int		node;		/* NUMA node */
 };
+extern struct pci_sysdata pci_default_sysdata;
 
 #ifdef CONFIG_PCI_DOMAINS
 static inline int pci_domain_nr(struct pci_bus *bus)
